.github/CODEOWNERS

```diff
@@ -6,3 +6,10 @@
 # be identified in the format @org/team-name. Teams must have
 # explicit write access to the repository. In this example,
 # the octocats team in the octo-org organization owns all .md files.
+
+###
+# Team Specific
+###
+
+# @bigcommerce/team-catalog
+reference/catalog @bigcommerce/team-catalog
```

Comment:
@olysenko we're not to the point of assigning domain-specific codeowners. please remove.

Reply:
OK. I will remove this change.